### DISCUSSION TITLE
Homebrew issue on M1

### DISCUSSION THREAD

**Discussion by carlca** [Homebrew/discussions#3078] [Everyday usage]
I am trying to install a package on macOS 12.2 Monterey on a M1 Mac Mini. The instructions are...
```
brew install librearp/librearp/librearp
update-librearp
```
but the first command falls over with this message...
```
Error: Cannot install in Homebrew on ARM processor in Intel default prefix (/usr/local)!
Please create a new installation in /opt/homebrew using one of the
"Alternative Installs" from:
  https://docs.brew.sh/Installation
You can migrate your previously installed formula list with:
  brew bundle dump
```
I've looked at `https://docs.brew.sh/Installation` as instructed but it doesn't really seem to offer any useful course of action.

Any clues, anyone? Thx!

**Comment by SMillerDev** [comment]
You've installed Homebrew with Rosetta so it's installed in `/usr/local`. Now you're running it without Rosetta so it's complaining that it doesn't have a Homebrew install in the ARM location (`/opt/homebrew`). You need to either run the installer again without Rosetta, or run the `brew install librearp/librearp/librearp` with Rosetta.

**Comment by carlca** [comment]
Not having much that cannot be reinstalled, I uninstalled the Rosetta version of Homebrew, and reinstalled it, this time into `/opt/homebrew` and everything is fine. I'm not sure how I came to install the Rosetta version of Homebrew, but all is well now 😉

**Reply by gromgit** [reply]
One of the following is true:
1. You installed Homebrew itself from a Rosetta-enabled terminal (all the paths that the installer showed you during the install would've started with `/usr/local`).
2. You restored Homebrew from an Intel Mac backup, or migrated to your current M1 Mac.
